issue_credential/issue_credential/v1_0/routes.py

```diff
@@ -0,0 +1,1654 @@
+
+    return web.json_response(invi_rec.serialize())
+
+
+async def register(app: web.Application):
+    """Register routes."""
+    app.add_routes(
+        [
+            web.get(
+                "/issue-credential/records", credential_exchange_list, allow_head=False
+            ),
+            web.post(
+                "/issue-credential/create-offer", credential_exchange_create_free_offer
+            ),
+            web.get(
+                "/issue-credential/records/{cred_ex_id}",
+                credential_exchange_retrieve,
+                allow_head=False,
+            ),
+            web.post("/issue-credential/create", credential_exchange_create),
+            web.post("/issue-credential/send", credential_exchange_send),
+            web.post(
+                "/issue-credential/send-proposal", credential_exchange_send_proposal
+            ),
+            web.post("/issue-credential/send-offer", credential_exchange_send_free_offer),
+            web.post(
+                "/issue-credential/records/{cred_ex_id}/send-offer",
+                credential_exchange_send_bound_offer,
+            ),
+            web.post(
+                "/issue-credential/records/{cred_ex_id}/send-request",
+                credential_exchange_send_request,
+            ),
+            web.post(
+                "/issue-credential/records/{cred_ex_id}/issue",
+                credential_exchange_issue,
+            ),
+            web.post(
+                "/issue-credential/records/{cred_ex_id}/store",
+                credential_exchange_store,
+            ),
+            web.post(
+                "/issue-credential/records/{cred_ex_id}/problem-report",
+                credential_exchange_problem_report,
+            ),
+            web.delete(
+                "/issue-credential/records/{cred_ex_id}",
+                credential_exchange_remove,
+            ),
+        ]
+    )
+
+    # OOB backwards compatibility. This is needed for adding v1 attachments.
+    for r in app.router.routes():
+        if r.resource and r.resource.canonical == "/out-of-band/create-invitation":
+            LOGGER.info(
+                "Replacing /out-of-band/create-invitation route for v1.0 capability"
+            )
+            r._handler = invitation_create
+
+
+def post_process_routes(app: web.Application):
+    """Amend swagger API."""
+    # Add top-level tags description
+    if "tags" not in app._state["swagger_dict"]:
+        app._state["swagger_dict"]["tags"] = []
+    app._state["swagger_dict"]["tags"].append(
+        {
+            "name": "issue-credential v1.0",
+            "description": "Credential issue v1.0",
+            "externalDocs": {"description": "Specification", "url": SPEC_URI},
+        }
+    )
+
+
+def register_events(bus: EventBus):
+    """Register event listeners."""
+    bus.subscribe(re.compile(r"^acapy::cred-revoked$"), cred_revoked)
+
+
+async def cred_revoked(profile: Profile, event: EventWithMetadata):
```

Comment:
This is the cred_ex revocation for v1 part. Aca-py has the same thing targeting the IssuerV2CredRevRecord's.